squidb-addons/squidb-sqlite-bindings/jni/sqlite/Android.mk

```diff
@@ -5,6 +5,7 @@ include $(CLEAR_VARS)
 # If using SEE, uncomment the following:
 # LOCAL_CFLAGS += -DSQLITE_HAS_CODEC
 LOCAL_CFLAGS += -DSQLITE_ENABLE_FTS3
+LOCAL_CFLAGS += -DSQLITE_ENABLE_JSON1
```

Comment:
Right now we're building this addon for all possible architectures, which is probably overkill. Let's distribute artifacts with binaries only for the most common architectures to reduce APK size for this addon.